qml/DisplaySettings.qml

```diff
@@ -65,21 +67,15 @@ Item {
 
                     Text {
                         text: "Theme:"
-                        color: Style.textColor
+                        color: Material.primary
                     }
 
                     ComboBox {
                         id: themeCB
                         Layout.columnSpan:  4
                         Layout.fillWidth: true
                         model: ["Dark", "Light"]
-                        onCurrentTextChanged: {
-                            if (currentIndex) {
-                                Style.useLightStyle()
-                            } else {
-                                Style.useDarkStyle()
-                            }
-                        }
+                        onCurrentTextChanged: StyleManager.isDark = !currentIndex
```

Comment:
Why this is wrong ?

Reply:
well, first because you are processing the event as a single 'toggling',
then you are using boolean operators to toggle it between 1 and 0 ...

wouldn't work if you decided to add, for instance, a third colorblind mode
I don't think it is going to cause any trouble, though.